roles/oneview_user/README.md

```diff
@@ -0,0 +1,33 @@
+## oneview_user
+Manage OneView Users.
+
+#### Synopsis
+ Provides an interface to manage Users. Can create, update, and delete.
+
+#### Requirements (on the host that executes the module)
+  * python >= 3.4.2
+  * hpeOneView >= 5.0.0
```

Comment:
Update oneview version

Reply:
Acutally it supports for >5.0.0

Reply:
The user resource will support for ov 5.0.0 (from V1200).